### ISSUE TITLE
produces different results than CLI with eslint-import-resolver-typescript

### ISSUE THREAD

**Issue by kasvtv** [microsoft/vscode-eslint#910]
In the following repository, viewing file `src/index.ts` with the extension enabled will show an error
But, running `npx eslint src/index.ts` will not!

Something is going wrong with the way the vscode extension treats this plugin

https://github.com/kasvtv/js-lint-format-boilerplate

VScode version: 1.42.1
Extension version: 2.1.1

**Comment by dbaeumer** [comment]
@kasvtv I actually see errors when running eslint using the version installed in the repository

![capture](https://user-images.githubusercontent.com/1931590/76203166-7ae68c00-61f6-11ea-903e-4d406b2b567b.png)

**Comment by dbaeumer** [comment]
Please ping if this doesn't solve your problem.

**Comment by kasvtv** [comment]
@dbaeumer Those are warnings, not errors :). I added an `eslint-disable` statement for this specific rule now.

Now the CLI reports nothing and vscode just reports `import/no-unresolved`.

**Comment by dbaeumer** [comment]
I don't see any errors in VS Code itself:

![capture](https://user-images.githubusercontent.com/1931590/76298306-8949ac00-62b9-11ea-91c2-bc2753aa8354.png)

**Comment by kasvtv** [comment]
That is weird, I do!
![eslint](https://user-images.githubusercontent.com/1628729/76613869-438e1d00-651f-11ea-8734-fb261b6ebe7a.png)

**Comment by kasvtv** [comment]
That's weird, I do!
![eslint](https://user-images.githubusercontent.com/1628729/76615097-a385c300-6521-11ea-876a-988d3e61b71f.png)

```json
{
  "breadcrumbs.enabled": false,
  "breadcrumbs.filePath": "off",
  "debug.inlineValues": true,
  "debug.toolBarLocation": "docked",
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "editor.fontSize": 14,
  "editor.formatOnSave": true,
  "editor.lineHeight": 17,
  "editor.matchBrackets": "always",
  "editor.minimap.enabled": false,
  "editor.tabSize": 2,
  "editor.trimAutoWhitespace": false,
  "editor.wrappingIndent": "indent",
  "eslint.alwaysShowStatus": true,
  "eslint.enable": true,
  "eslint.workingDirectories": [
    {
      "directory": "."
    }
  ],
  "extensions.ignoreRecommendations": true,
  "extensions.showRecommendationsOnlyOnDemand": true,
  "files.exclude": {
    ".github": true,
    ".idea": true,
    ".vscode": true,
    "**/.DS_Store": false,
    "**/.hg": false,
    "**/.svn": false,
    "**/CVS": false,
    "**/node_modules": true
  },
  "telemetry.enableCrashReporter": false,
  "telemetry.enableTelemetry": false,
  "terminal.integrated.shell.windows": "C:\\Program Files\\Git\\bin\\bash.exe",
  "workbench.colorCustomizations": {
    "editorIndentGuide.background": "#ffffff0d",
    "editorIndentGuide.activeBackground": "#ffffff0d"
  },
  "workbench.editor.labelFormat": "medium",
  "workbench.enableExperiments": false,
  "workbench.iconTheme": "material-icon-theme",
  "workbench.startupEditor": "none",
  "workbench.tree.indent": 20,
  "javascript.suggestionActions.enabled": false,
  "[javascript]": {
    "editor.semanticHighlighting.enabled": false,
  }
}
```

**Comment by dbaeumer** [comment]
Sorry, still not able. Could you try to tweak the sample so that it reproduce out of the box (without relying on global installed modules / settings)
